app/assets/stylesheets/application.scss

```diff
@@ -78,6 +78,14 @@ a:hover, a:focus {
   border-color: $white;
 }
 
+.review-info p {
+  position: relative;
```

Comment:
Properties should be ordered border-color, border-right, position, top, transform